source/includes/getting-started/_concepts.html.md

```diff
@@ -2,29 +2,29 @@
 
 ![The Bonsai Platform][1]
 
 ###### Build
 
 * **Bonsai Command Line Interface (CLI):** is a command line tool that enables you to configure and control the Bonsai Artificial Intelligence (AI) Engine.
-* **beta.bons.ai:** The address for Bonsai's beta web interface; this is the browser based tool that includes your *BRAIN Dashboard* and *BRAIN Details* page.
+* **[beta.bons.ai][4]:** The address for Bonsai's beta web interface; this is the browser based tool that includes your *BRAIN Dashboard* and *BRAIN Details* page.
 * **Simulators:** are currently the only training source implemented. Simulators are often a complex, interactive virtual environment. They usually have state, which is a representation of the world inside the virtual environment.
 * **Generators:** are a training source. They produce labeled data programmatically. This data is effectively infinite. A generator could, for example, produce a random (but known) integer, set of line segments, etc.
-* **Data:** is also a training source. It is information related to the scenario being trained, or to test and evaluate how well the training is implemented. Non-generated data may sometimes be difficult to find, and the data that exists may be a very small set or difficult to label. An example of a data set you could train with is the [MNIST database][5], which is a collection of handwritten digits used for training image processing systems and other machine learning systems.
+* **Data:** is also a training source. It is information related to the scenario being trained and can also be used to test and evaluate how well the training is implemented. Data could be a collection of images and labels or the rows of a spreadsheet. An example of a data set you could train with is the [MNIST database][5], which is a collection of handwritten digits used for training image processing systems and other machine learning systems.
 
 **Note:** Currently, during our private beta, you can only use simulators as your training source.
 
 [//]: # (Simulators are the only training source, beta.bons.ai is current website instead of brains.bons.ai)
 
 ###### Teach
 
-* **Inkling:** our unique, special purpose, pedagogical programming language. Pedagogical programming is a new programming paradigm. It expresses the solution to a problem in terms of how you teach the computer to find the solution as opposed to how to calculate the solution.
+* **Inkling:** our unique, special purpose, pedagogical programming language. Pedagogical programming is a new teaching programming paradigm. It expresses the solution to a problem in terms of how you teach the computer to find the solution as opposed to how to calculate the solution.
```

Comment:
Pedagogical programming is a "programming paradigm" not a "teaching programming paradigm". I would revert this change.